tests/integration/components/frost-icon-test.js

```diff
@@ -10,21 +18,47 @@ describeComponent(
   },
 
   function () {
-    it('renders', function () {
-      // Set any properties with this.set('myProperty', 'value');
-      // Handle any actions with this.on('myAction', function(val) { ... });
-      // Template block usage:
-      // this.render(hbs`
-      //   {{#frost-icon}}
-      //     template content
-      //   {{/frost-icon}}
-      // `);
+    beforeEach(function () {
+      initialize()
+    })
+
+    it('renders default values', function () {
+      this.render(hbs`
+        {{frost-icon icon='round-add'}}
+      `)
+
+      expect(
+        this.$('.frost-icon').hasClass('frost-icon-frost-round-add'),
+        'Icon class is set correctly.'
+      ).to.be.true
+    })
+
+    it('sets the pack property', function () {
+      this.render(hbs`
+        {{frost-icon
+          pack='test'
+          icon='round-add'
+        }}
+      `)
+
+      expect(
+        this.$('.frost-icon').hasClass('frost-icon-test-round-add'),
+        'Icon class is set correctly.'
+      ).to.be.true
+    })
 
+    it('sets the hook property', function () {
       this.render(hbs`
-        {{frost-icon icon='frost/round-add'}}
+        {{frost-icon
+          hook='my-icon'
```

Comment:
This is a good point. In this case since we are not doing any concatenation or anything else with `ember-hook` we are not doing anything with this test other than testing that `ember-hook` is doing what it should. We should remove this test.